src/main/java/org/opentripplanner/graph_builder/module/GtfsModule.java

```diff
@@ -217,8 +297,18 @@ private GtfsMutableRelationalDao loadBundle(GtfsBundle gtfsBundle)
                         reader.addAgencyIdMapping(agencyId, generatedAgencyId); // NULL key should work
                         agency.setId(generatedAgencyId);
                         agencyId = generatedAgencyId;
+                        agencyIdIsAutogenerated = true;
+                    } else {
+                        // the agencyId was not null or was non-conflicting. This means that either the agencyId was
+                        // derived from a record in the agency.txt table, or it was assigned the default AgencyId which
+                        // is the feedID of this feed.
```

Comment:
I think I understand what this is trying to say. I think the explanation should state how it is related to `agencyFileContainedOneAgencyWithoutId`